ote/src/clj/ote/transit_changes/detection.clj

```diff
@@ -456,7 +463,9 @@
           (conj results filtered-diff-data))
         (if (empty? results)
           diff-data
-          results)))))                                      ;; Default week data expected when no changes are found
+          (if (some? filtered-diff-data)
+            (conj results filtered-diff-data)
```

Comment:
Is this ok or is there an issue? 
`filter` returns a list. `conj` prepends to lists, but append is expected here?

Reply:
FIltered diff data is a map, Not a list